Frameworks/UIKit/UILabel.mm

```diff
@@ -388,45 +317,43 @@ - (void)setText:(NSString*)newStr {
     }
     if (newStr == nil || ![_text isEqual:newStr]) {
         _text = [newStr copy];
-
-        if (_adjustFontSize) {
-            [self _adjustFontSizeToFit];
-        } else {
-            [self _updateXamlElement];
-        }
-
+        [self _adjustFontSizeForXamlElement];
         self.accessibilityValue = newStr;
     }
 }
 
 /**
- @Status Stub
+  @Status NotInPlan
+  @Notes Attributed text is not easily supported by Xaml.
 */
 - (void)setAttributedText:(NSAttributedString*)newStr {
-    UNIMPLEMENTED();
+    UNIMPLEMENTED_WITH_MSG("Attributed text is not easily supported by Xaml.");
     _attributedText = [newStr copy];
     [self setText:[_attributedText string]];
 }
 
 /**
- @Status Stub
+  @Status NotInPlan
+  @Notes Attributed text is not easily supported by Xaml.
 */
 - (NSAttributedString*)attributedText {
-    UNIMPLEMENTED();
+    UNIMPLEMENTED_WITH_MSG("Attributed text is not easily supported by Xaml.");
     return _attributedText;
 }
 
 /**
- @Status Interoperable
+ @Status Stub
 */
 - (void)setEnabled:(BOOL)enable {
+    UNIMPLEMENTED();
     _isDisabled = !enable;
 }
 
 /**
- @Status Interoperable
+ @Status Stub
 */
```

Comment:
Let's go with NotInPlan because this isn't a property that XAML has available for TextBlock.  We can reconsider if customers need this and manually dim the text when it's disabled.